lua/core/plugins.lua

```diff
@@ -286,11 +286,10 @@ if packer_status_ok then
 
     -- Start screen
     {
-      "glepnir/dashboard-nvim",
+      "goolord/alpha-nvim",
       config = function()
-        require("configs.dashboard").config()
+        require("configs.alpha").config()
       end,
-      disable = not config.enabled.dashboard,
```

Comment:
It would be nice if you can make alpha can be lazy-loaded like current dashboard.

Reply:
This can be done with the `init.lua`:

```lua
return {
  plugins = {
    init = {
      ["goolord/alpha-nvim"] = { disable = true },
    },
  },
}
```